backend/src/server/routes/v3/login-router.ts

```diff
@@ -42,7 +42,8 @@ export const registerLoginRouter = async (server: FastifyZodProvider) => {
     },
     schema: {
       body: z.object({
-        organizationId: z.string().trim()
+        organizationId: z.string().trim(),
+        customUserAgent: z.enum(["cli"]).optional()
```

Comment:
nit: might be better called just `userAgent`. If they are passing this in then we can assume it is custom